bibigrid-core/src/main/resources/playbook/roles/master/tasks/002-zabbix.yml

```diff
@@ -78,21 +77,93 @@
 # -- Zabbix Web Frontend --
 # -------------------------
 
-- name: Debian based OS
+- name: Zabbix Web Frontend related packages for Debian (Debian 10, Ubuntu 18.04 and Ubuntu 20.04) based OS
   block:
-    - name: Install apache web server
+    - name: Install apache web server and php
       apt:
-        name: apache2
+        name:
+          - apache2
+          - php
         state: latest
 
     - name: Install zabbix web frontend
       apt:
-        name: zabbix-frontend-php
+        name:
+          - zabbix-frontend-php
+          - zabbix-apache-conf
         state: latest
-  when: "ansible_distribution_file_variety == 'Debian'"
+  when: "ansible_distribution_file_variety == 'Debian' and not ( ansible_distribution == 'Ubuntu' and ansible_distribution_release == 'xenial' )"
 
 
-- name: REHL based OS
+- name: Zabbix Web Frontend related packages for Ubuntu 16.04
+  block:
+    - name: Install apache web server and php 7.4
+      apt:
+        name:
+          - apache2
+          - php7.4
+          - php7.4-bcmath
+          - php7.4-bz2
+          - php7.4-cgi
+          - php7.4-cli
+          - php7.4-common
+          - php7.4-curl
+          - php7.4-dba
+          - php7.4-dev
+          - php7.4-enchant
+          - php7.4-fpm
+          - php7.4-gd
+          - php7.4-gmp
+          - php7.4-imap
+          - php7.4-interbase
+          - php7.4-intl
+          - php7.4-json
+          - php7.4-ldap
+          - php7.4-mbstring
+          - php7.4-mysql
+          - php7.4-odbc
+          - php7.4-opcache
+          - php7.4-pgsql
+          - php7.4-phpdbg
+          - php7.4-pspell
+          - php7.4-readline
+          - php7.4-snmp
+          - php7.4-soap
+          - php7.4-sqlite3
+          - php7.4-sybase
+          - php7.4-tidy
+          - php7.4-xml
+          - php7.4-xmlrpc
+          - php7.4-xsl
+          - php7.4-zip
+          - libapache2-mod-php7.4
+          - libphp7.4-embed
+        state: latest
+
+    - name: Install zabbix web frontend
+      apt:
+        name:
+          - zabbix-frontend-php-deprecated
+        state: latest
```

Comment:
Seems to refer to [Zabbix 5.0 Documentation](https://www.zabbix.com/documentation/current/manual/installation/install_from_packages/frontend_on_debian) with 'zabbix-frontend-php-deprecated [...] A web server as well as PHP 7.2 with its modules have to be installed manually (use PPAs / build PHP from source).'
The site provides required php modules for Zabbix frontend: 
- php-gd
- php-bcmath
- php-mbstring
- php-xml
- php-ldap
- php-json
You may know better, if the other modules are also important to install

Reply:
Good point! When testing different PHP versions on Ubuntu 16.04 I do always a full install, installing everything coming with the PHP ppa. Time to  check the dependencies needed by Zabbix.

Reply:
Fixed in #228